kornia/transpiler/transpiler.py

```diff
@@ -0,0 +1,80 @@
+import ivy
```

Comment:
I think we may use the `LazyLoader` module for this, so that we do not require user to have `ivy` if they do not want to use it.

You may have
```python
ivy = LazyLoader("ivy")
```
in
https://github.com/kornia/kornia/blob/9ef40649affa59222215afa11e66381bf874b4a3/kornia/core/external.py#L94-L95

Then, to use it, we use
```python
from kornia.core.external import ivy
```

Reply:
:+1: